node.gyp

```diff
@@ -8,6 +8,7 @@
     'node_has_winsdk%': 'false',
     'node_shared_zlib%': 'false',
     'node_shared_http_parser%': 'false',
+    'node_shared_libcares%': 'false',
```

Comment:
We previously called it `node_shared_cares` and it might be nice to keep that name in the (admittedly unlikely) case that people for `process.config.variables.node_shared_cares`.